spec/numberformat.html

```diff
@@ -215,6 +219,7 @@ <h1>
         1. Else,
           1. If IsWellFormedCurrencyCode(_currency_) is *false*, throw a *RangeError* exception.
         1. Let _currencyDisplay_ be ? GetOption(_options_, *"currencyDisplay"*, ~string~, &laquo; *"code"*, *"symbol"*, *"narrowSymbol"*, *"name"* &raquo;, *"symbol"*).
+        1. Let _currencyPrecision_ be ? GetOption(_options_, *"currencyPrecision"*, ~string~, &laquo; *"cash"*, *"financial"* &raquo;, *"financial"*).
```

Comment:
Just a reminder to bikeshed all parts of this: the option name and both string options.

Reply:
My sense is "no", since there's situations where you'd want to use one and not the other. I can easily imagine situations where someone using (for example) a personal finance application would want debts represented by wrapping them in parentheses, but not particularly care about sub-cent values. 

Avoiding confusion between the options is one reason why I think "financial" would be a better name than "accounting" for this option.